cli/package.json

```diff
@@ -33,8 +33,8 @@
     "start": "node dist/index.js",
     "lint": "eslint . --report-unused-disable-directives",
     "lint:fix": "pnpm lint --fix",
-    "format": "prettier --write --plugin-search-dir=. **/*.{cjs,mjs,ts,tsx,md,json} --ignore-path ../.gitignore --ignore-unknown --no-error-on-unmatched-pattern",
-    "format:check": "pnpm format --check",
+    "format": "pnpm format:check --write",
```

Comment:
does `--write` override the `--check` in the `:check` script?

bit cursed but i'll allow it

Reply:
<img width="1328" alt="image" src="https://user-images.githubusercontent.com/34718806/236797378-49639c3a-f9bf-43d9-a6d4-2d71fcff7778.png">
P much